src/platform/common/platform/fileSystem.ts

```diff
@@ -40,8 +40,8 @@ export class FileSystem implements IFileSystem {
 
     // URI-based filesystem functions
 
-    async copy(source: vscode.Uri, destination: vscode.Uri): Promise<void> {
-        await this.vscfs.copy(source, destination);
+    async copy(source: vscode.Uri, destination: vscode.Uri, options?: { overwrite: boolean }): Promise<void> {
+        await this.vscfs.copy(source, destination, options);
```

Comment:
Both fs and VS Code support `options` and we were't passing them, but was hardcoded in the node implementation